docs/content/configuration/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Configuring Athens"
 date: 2018-10-16T12:14:01-07:00
+chapter: true
```

Comment:
Please remove this value, chapter templates are styled with text content centered. So while this is ok for chapter index pages with just a title and/or subtitle it makes it hard to read for more verbose pages like this. See the netlify deploy preview to see what I'm talking about.

Reply:
https://deploy-preview-883--gomods-athens.netlify.com/

Reply:
done